File: 1_Periodo/Lista 5/Python/L5_Q8.py
valorHora = float(input("Informe o valor da hora de trabalho: "))
hora = int(input("Informe quantas horas trabalhadas no mês: "))

salario = valorHora*hora
salario += salario*0.1

print(f"O valor a ser pago para o funcionario é R$ {salario: .2f}")

File: 1_Periodo/Lista 5/C/L5_Q8.c
#include <stdio.h>

int main(){
	
	float valor_hora, horas, valor_pago;
	
	printf("Informe o valor da hora de trabalho (em reais): ");
	scanf("%f", &valor_hora);
	
	printf("Informe o numero de horas trabalhadas no mes: ");
	scanf("%f", &horas);
	
	valor_pago = valor_hora * horas;
	valor_pago += valor_pago * 0.1;
	
	printf("Valor a ser pago eh R$%.2f", valor_pago);
}
